### PR TITLE
Added missing insert stmts in key_policy table

### PR DESCRIPTION
**Pull request by ase-101**

### CODE REVIEW

File: db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql [hunk 1]
```diff
@@ -9,6 +9,11 @@ ALTER TABLE keymgr.key_alias ADD CONSTRAINT uni_ident_const UNIQUE (uni_ident);
 ALTER TABLE keymgr.key_policy_def ADD COLUMN pre_expire_days smallint;
 ALTER TABLE keymgr.key_policy_def ADD COLUMN access_allowed character varying(1024);
 
+insert into keymgr.key_policy_def(app_id, key_validity_duration, is_active, cr_by, cr_dtimes, pre_expire_days, access_allowed) values('ADMIN_SERVICES',1095,TRUE,'mosipadmin',now(),60,'NA');
+insert into keymgr.key_policy_def(app_id, key_validity_duration, is_active, cr_by, cr_dtimes, pre_expire_days, access_allowed) values('RESIDENT',1095,TRUE,'mosipadmin',now(),60,'NA');
+insert into keymgr.key_policy_def(app_id, key_validity_duration, is_active, cr_by, cr_dtimes, pre_expire_days, access_allowed) values('DIGITAL_CARD',1095,TRUE,'mosipadmin',now(),60,'NA');
```
Comment (vishwa-vyom) [inline db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql:14]: As part of https://github.com/mosip/keymanager/blob/1.1.5.5/db_scripts/mosip_keymgr/dml/keymgr-key_policy_def.csv I see DIGITAL_CARD already, do we need to insert this ?

File: db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql [hunk 2]
```diff
@@ -17,6 +22,4 @@ update keymgr.key_policy_def set pre_expire_days=60, access_allowed='NA' where a
 update keymgr.key_policy_def set pre_expire_days=60, access_allowed='NA' where app_id='KERNEL';
 update keymgr.key_policy_def set pre_expire_days=1125, access_allowed='NA' where app_id='ROOT';
 update keymgr.key_policy_def set pre_expire_days=30, access_allowed='NA' where app_id='BASE';
-update keymgr.key_policy_def set pre_expire_days=395, access_allowed='NA' where app_id='PMS';
```
Comment (vishwa-vyom) [inline db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql:20]: Why is PMS removed from here ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql (hunk 2 above)
Comment (ase-101) [reply db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql:20]: It is not removed, because of the new line it moved below.

Comment (vishwa-vyom) [reply db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql:20]: oh ok

**Review by vishwa-vyom — APPROVED** [review]

File: db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql (hunk 1 above)
Comment (ase-101) [reply db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql]: Its added in 1.1.5.5 on july 1st, seems to be a recent change there. 'DIGITAL_CARD' is added on the released branch.

**Pull request merged**